components/file_system/src/lib.rs

```diff
@@ -464,14 +464,12 @@ pub fn trash_dir_all(path: impl AsRef<Path>) -> io::Result<()> {
     remove_dir_all(trash_path)
 }
 
-/// When using `trash_dir_all`, it's possible the directory is marked as trash
-/// but not being actually deleted after a restart. This function can be used
-/// to resume all those removal in the given directory.
+// remove all file exclude prefix.
 #[inline]
-pub fn clean_up_trash(path: impl AsRef<Path>) -> io::Result<()> {
```

Comment:
This method is supposed to be used paired with `trash_dir_all`.

Reply:
yes fixed it with one parameter `prefix`

Reply:
You should keep this method and introduce a new method instead. You may make this method call the new method.